db/seeds/data/state_benefit_types.yml

```diff
@@ -3,75 +3,199 @@
-  exclude_from_gross_income: true
+  :name: Fostering Allowance (to the extent it exceeds dependents allowance)
+  :exclude_from_gross_income: true
+  :dwp_code:
+grenfell_payments:
+  :name: Grenfell Payments
+  :exclude_from_gross_income: true
+  :dwp_code:
 housing_benefit:
-  name: Housing Benefit
-  exclude_from_gross_income: true
+  :name: Housing Benefit
+  :exclude_from_gross_income: true
+  :dwp_code:
+incapacity_benefit:
+  :name: Incapacity Benefit
+  :exclude_from_gross_income: false
+  :dwp_code: IB
+income_support:
+  :name: Income Support
+  :exclude_from_gross_income: false
+  :dwp_code: IS
 independent_living_funds_payments:
-  name: Independent Living Funds payments
-  exclude_from_gross_income: true
-welsh_independent_living_grant:
-  name: Welsh Independent Living Grant
-  exclude_from_gross_income: true
+  :name: Independent Living Funds payments
+  :exclude_from_gross_income: true
+  :dwp_code:
+industrial_injuries_benefit:
+  :name: Industrial Injuries Benefit
+  :exclude_from_gross_income: false
+  :dwp_code: IIB
+industrial_injuries_disablement_benefit:
+  :name: Industrial Injuries Disablement Benefit
+  :exclude_from_gross_income: false
+  :dwp_code: IIDB
+jobseekers_allowance:
+  :name: Jobseeker’s Allowance
+  :exclude_from_gross_income: false
+  :dwp_code: JSA
+lump_sum_payments_under_windows_pension:
+  :name: Lump sum payments under Window's Pension
+  :exclude_from_gross_income: true
+  :dwp_code:
+maternity_allowance:
+  :name: Maternity Allowance
+  :exclude_from_gross_income: false
+  :dwp_code: MA
+minimum_income_guarantee:
+  :name: Minimum Income Guarantee
+  :exclude_from_gross_income: false
+  :dwp_code: MIG
+mortgage_interest_direct:
+  :name: Mortgage Interest Direct
+  :exclude_from_gross_income: false
+  :dwp_code: MID
 payments_on_account_of_benefit:
-  name: Payments on Account of Benefit
-  exclude_from_gross_income: true
+  :name: Payments on Account of Benefit
+  :exclude_from_gross_income: true
+  :dwp_code:
+pension_credit:
+  :name: Pension Credit
+  :exclude_from_gross_income: false
+  :dwp_code: PC
 personal_independent_payments:
-  name: Personal Independent Payments
-  exclude_from_gross_income: true
-armed_forces_independence_payment:
-  name: Armed Forces Independence Payment
-  exclude_from_gross_income: true
+  :name: Personal Independent Payments
+  :exclude_from_gross_income: true
+  :dwp_code: PIP
+reduced_earnings_allowance:
+  :name: Reduced Earnings Allowance
+  :exclude_from_gross_income: false
+  :dwp_code: REA
 severe_disablement_allowance:
-  name: Severe Disablement Allowance
-  exclude_from_gross_income: true
+  :name: Severe Disablement Allowance
+  :exclude_from_gross_income: true
+  :dwp_code: SDA
+social_fund:
+  :name: Social Fund
+  :exclude_from_gross_income: false
+  :dwp_code: SF
 social_fund_payments:
-  name: Social Fund Payments
-  exclude_from_gross_income: true
+  :name: Social Fund Payments
+  :exclude_from_gross_income: true
+  :dwp_code:
```

Comment:
Are Social Fund Payments known to be different from Social Fund? Should this be here without a code?

Reply:
Yes it may be the same thing.  This is an amalgamation of two lists of benefits: one from the DWP with codes, the other from the LAA without codes.  The LAA may be slightly different wording for one of the DWP ones -it may be an entirely different benefit that isn't in the DWP list.

So we're going to use this table to identify ones with codes automatically from the bank statement.  We can also identify other DWP payments, and will ask the applicant about them and record the response.  We may be able in time to determine the codes for other benefits.

Dave is going to do a bit more research into the names the LAA use.